package.json

```diff
@@ -26,18 +26,19 @@
     }
   ],
   "devDependencies": {
-    "chai": "^1.10.0",
-    "dateformat": "^1.0.11",
-    "eslint": "^2.2.0",
-    "eslint-config-eslint": "^3.0.0",
-    "eslint-release": "^0.3.0",
-    "istanbul": "~0.2.6",
-    "leche": "^1.0.1",
-    "mocha": "^2.0.1",
-    "npm-license": "^0.2.3",
-    "semver": "^4.1.1",
-    "shelljs": "^0.3.0",
-    "shelljs-nodecli": "^0.1.1"
+    "chai": "3.5.0",
+    "dateformat": "1.0.11",
+    "eslint": "2.2.0",
+    "eslint-config-eslint": "3.0.0",
+    "eslint-release": "0.3.0",
+    "istanbul": "0.4.2",
+    "leche": "2.1.1",
+    "mocha": "2.0.1",
+    "npm-license": "0.3.1",
+    "semver": "5.1.0",
+    "shelljs": "0.6.0",
+    "shelljs-nodecli": "0.1.1",
+    "typescript": "1.8.7"
```

Comment:
using npm 3 it is not getting installed for development
and not changing npm module
so it is nice to have